taxy.el

```diff
@@ -390,10 +390,10 @@ defined with a definer defined by `taxy-define-key-definer')."
                              :name "Numbers"
                              :taxys (list (make-taxy
                                            :name "Odd"
-                                           :predicate #'cl-oddp)
+                                           :predicate #'cl-cl-oddp)
```

Comment:
Oops.  :)  (There are 3 others of these, too.)

Reply:
Agh...  I goofed the `my/project-consult-git-grep > wgrep` workflow.  Thanks for catching my mistake!

Reply:
Fixed in latest force push.